### PR TITLE
[plugin.video.skyvideoitalia] 1.0.4

### PR DESCRIPTION
**Pull request by nixxo**
### Description:
Update to the SkyVideo Italia plugin.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.skyvideoitalia/resources/lib/addonutils.py [hunk 1]
```diff
@@ -78,19 +79,15 @@ def setSetting(setting, value):
...
         label='', params=None, label2=None,
-        thumb=None, fanart=None, poster=None, arts=None,
-        videoInfo=None, properties=None, isFolder=True,
+        thumb=None, fanart=None, poster=None, arts={},
+        videoInfo=None, properties={}, isFolder=True,
         path=None, subs=None):
-    if arts is None:
-        arts = {}
-    if properties is None:
-        properties = {}
```
Comment (enen92) [inline plugin.video.skyvideoitalia/resources/lib/addonutils.py:93]: While on it maybe add offscreen=True  to bypass guilocks

File: plugin.video.skyvideoitalia/resources/lib/addonutils.py [hunk 2]
```diff
@@ -128,23 +125,18 @@ def addListItem(
...
 def setResolvedUrl(
-        url='', solved=True, subs=None, headers=None,
-        ins=None, insdata=None, item=None, exit=True):
+        url='', solved=True, headers=None, subs=None,
+        item=None, exit=True):
     headerUrl = ''
     if headers:
         headerUrl = urlencode(headers)
-    item = xbmcgui.ListItem(path=url + '|' + headerUrl) if item is None else item
+    item = xbmcgui.ListItem(
```
Comment (enen92) [inline plugin.video.skyvideoitalia/resources/lib/addonutils.py:133]: Same

**Review by enen92 — APPROVED** [review]
LGTM, some suggestions if you want to speed things a bit

File: plugin.video.skyvideoitalia/resources/lib/addonutils.py (hunk 1 above)
Comment (nixxo) [reply plugin.video.skyvideoitalia/resources/lib/addonutils.py:93]: done.

File: plugin.video.skyvideoitalia/resources/lib/addonutils.py (hunk 2 above)
Comment (nixxo) [reply plugin.video.skyvideoitalia/resources/lib/addonutils.py:133]: done.

**Pull request merged**